data/mods/TropiCataclysm/monsters/tropical_fish.json

```
@@ -169,5 +205,54 @@
     "copy-from": "mon_fish_tiny",
     "name": { "str": "angelfish", "str_pl": "angelfishes" },
```

```suggestion
    "name": { "str_sp": "angelfish" },
```
I think `fishes` is only used to refer to multiple subspecies.